### DISCUSSION TITLE
Analogue recognition is off for one of the digits

### DISCUSSION THREAD

**Discussion by gaborbernat** [jomjol/AI-on-the-edge-device#2173] [Q&A]
<img width="872" alt="image" src="https://user-images.githubusercontent.com/690238/224606105-d15c894c-730c-4fa7-b78a-538ab20ef7e2.png">

Any recommendations to improve it? 🤔  You can see the third digit shows 1.0 even though is more like 9.

**Comment by Piti61** [comment]
Is "Counter-Clockwise Rotation" active?

**Reply by gaborbernat** [reply]
It's not but seems alignment sometimes puts image upside down 🤔

**Comment by caco3** [comment]
- Your contrast is far to low!
- Is it possible to make the image sharper (Adjust the focus)?
- Also the alignment fails because the marks don't have a good contrast.
  I suggest to put stickers on the glass and use them as marks, see https://jomjol.github.io/AI-on-the-edge-device-docs/Demo-Mode/#example-data-of-a-water-meter for an example.

**Reply by gaborbernat** [reply]
> * Your contrast is far to low!
> * Is it possible to make the image sharper (Adjust the focus)?

Can't get anything better 🤔 

The aligment works fine enough for now.

<img width="776" alt="image" src="https://user-images.githubusercontent.com/690238/226140570-25a45845-4500-4d68-9389-79044a2db352.png">

This is the clearest the picture gets with mopving the ov2640 focus...
